docs/configuration.asciidoc

```diff
@@ -22,7 +22,7 @@ secret_token: <%= ENV["VERY_SECRET_TOKEN"] %>
 Options are applied in the following order (last one wins):
 
 1. Defaults
-2. Arguments to `ElasticAPM.start` / `Config.new`
+2. Arguments to `ElasticAPM.start` / `ElasticAPM::Grape.start` / `ElasticAPM::Rails.start` / `Config.new`
```

Comment:
Could _Arguments to `.start(…)`_ be explanatory enough? I'm afraid mentioning too many different frameworks here will confuse more than help?